lightwood/encoder/image/img_2_vec.py

```diff
@@ -40,12 +56,20 @@ def prepare(self, train_priming_data: pd.Series, dev_priming_data: pd.Series):
         self.is_prepared = True
 
     def to(self, device, available_devices):
+        """
+        Moves the model to-and-from CPU and GPU.
+
+        :param device: will move the model to this device.
+        :param available_devices: all available devices as reported by lightwood.
+
+        :return: same object but moved to the target device.
+        """
         self.model.to(device, available_devices)
         return self
 
     def encode(self, images: List[str]) -> torch.Tensor:
         """
-        Encode list of images
+        Encode a list of images.
```

Comment:
May want to use the term "Creates an encoding for images; each image is referenced by the filepath or url"